crates/core_arch/src/x86/rtm.rs

```diff
@@ -0,0 +1,151 @@
+#[cfg(test)]
```

Comment:
Can you add a top-level comment here about these ?  E.g. to the spec, and maybe to a wikipedia page or some summary page explaining these ? Is for internal documentation only.

Reply:
Pushed